debian/changelog

```diff
@@ -1,3 +1,215 @@
+pulseaudio (1:6.0-1endless1) eos; urgency=low
```

Comment:
@mariospr For our other forked packages, we've been letting the endless changes just accumulate with a version calculated by the jenkins scripts. To that end, I think we can drop the `endless1` and just let the version be `1:6.0`. I could go either way, though. Regardless, we'll need to push a tag that matches whatever goes here.